common/tools/dev-tool/src/commands/run/bundle.ts

```diff
@@ -112,14 +123,28 @@ export default leafCommand(commandInfo, async (options) => {
           preferBuiltins: false,
           browser: true,
         }),
-        ...(options["polyfill-node"] ? [nodePolyfills({ sourceMap: true })] : []),
         cjs({
           dynamicRequireTargets: [globFromStore("chai")],
         }),
+
+        ...(options["inject-node-polyfills"]
+          ? [
+              inject({
+                modules: {
```

Comment:
do we need to make this list configurable? Not sure if all packages need all three.

Reply:
For context, this plugin injects corresponding import statements for occurrences of the keys of the modules option. For example, for `Buffer: ['buffer', 'Buffer']`, it will search for references for `Buffer` and for each one it will add `import { Buffer } from "buffer";`. So if there is no occurrence for some of those, no corresponding import statements will be added.

I think configuring would be useful if for some reason two libraries want to polyfill the same definition with two different libraries and I didn't bump into a situation like this yet so I opted for simplicity for now.